React/Views/RCTModalHostView.m

```diff
@@ -54,6 +61,27 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
   return self;
 }
 
+#if TARGET_OS_TV
+- (void)menuButtonPressed:(__unused UIGestureRecognizer *)r
+{
+    if(_onRequestClose) {
+        _onRequestClose(nil);
+    }
+}
+
+- (void)setOnRequestClose:(RCTDirectEventBlock)onRequestClose
+{
+  _onRequestClose = onRequestClose;
+  if([self.subviews count] > 0) {
+    if(_onRequestClose && _tapGestureRecognizer) {
+      [self.subviews[0] addGestureRecognizer:_tapGestureRecognizer];
```

Comment:
Oh, actually, `subviews` is always empty. So, it is exactly what I mean by fragile. 😞